api/provider/aws/apps_test.go

```diff
@@ -46,13 +46,13 @@ func TestAppGet(t *testing.T) {
 		},
 		Parameters: map[string]string{
 			"WebMemory":              "256",
+			"WebCpu":                 "256",
 			"Release":                "RVFETUHHKKD",
 			"Subnets":                "subnet-13de3139,subnet-b5578fc3,subnet-21c13379",
 			"Private":                "Yes",
 			"WebPort80ProxyProtocol": "No",
 			"VPC":                  "vpc-f8006b9c",
 			"Cluster":              "convox-Cluster-1E4XJ0PQWNAYS",
-			"Cpu":                  "200",
```

Comment:
Good catch! I now recall we played with an app-wide Cpu parameter before, but stopped doing that and went back to Cpu=0 everywhere.